### PR TITLE
COL-1435 Node 10 upgrade with dependencies

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/COL-1435

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,4 +19,4 @@ before_install:
   - cd ${TRAVIS_BUILD_DIR}
 
 script:
-  - node_modules/.bin/gulp travis
+  - node_modules/.bin/gulp test
```
Comment (johncrossman) [inline .travis.yml:22]: @paulkerschen - I think node version is in `.travis.yml`. For other projects, when I updated node I added:
```
before_install:
  - nvm install 12
```
Not sure if something like that is necessary for certain versions of Node.

**Comment by johncrossman** [comment]
@paulkerschen - Do we also need to update `ami.config`?  https://github.com/ets-berkeley-edu/suitec-preview-service/blob/master/.ebextensions/00_ami.config#L3

**Comment by paulkerschen** [comment]
@johncrossman Yes, thanks, I’ll update the ami.config.

File: .travis.yml (hunk 1 above)
Comment (paulkerschen) [reply .travis.yml:22]: I’ll update the version in travis.yml and see if that works.

**Comment by paulkerschen** [comment]
@johncrossman Looks like we had a successful (minimal) Travis run with Node 10.22.1 correctly set.

**Pull request merged**